esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -420,6 +420,64 @@ void QueryFilesInUse::loadTargets(IPropertyTree *t, unsigned flags)
     }
 }
 
+void QueryFilesInUse::updateUsers()
+{
+    SCMStringBuffer target;
+
+#ifdef _CONTAINERIZED
+    Owned<IStringIterator> targets = getContainerTargetClusters("roxie", nullptr);
+    ForEach(*targets)
+    {
+        Owned<IConstWUClusterInfo> clusterInfo = getWUClusterInfoByName(targets->str(target).str());
+        if (!clusterInfo)
+            return;
+
+        SCMStringBuffer secretKey, vaultId;
+        secretKey.set(clusterInfo->getFileSecretKey());
+        vaultId.set(clusterInfo->getFileVaultKey());
+
+        StringBuffer username, password;
+        if (secretKey.length() > 0)
+        {
+            Owned<IPropertyTree> secretTree;
+            if (vaultId.length() > 0)
+                secretTree.setown(getVaultSecret("authn", vaultId.str(), secretKey.str(), nullptr));
+            else
+                secretTree.setown(getSecret("authn", secretKey.str()));
+            if (!secretTree)
+                throw makeStringExceptionV(-1, "Error retrieving Roxie file secret for FileSecretKey %s.", secretKey.str());
+
+            getSecretKeyValue(username, secretTree, "username");
+            getSecretKeyValue(password, secretTree, "password");
```

Comment:
the password is not going to be used, the roxie (or any dali client, and in near future esp client with certificate) is implicitly trusted.